README.md

```diff
@@ -1,6 +1,12 @@
 # Cartography
 
-Nventive solution for mobile app map.
+[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg?style=flat-square)](LICENSE) ![Version](https://img.shields.io/nuget/v/Cartography.DynamicMap?style=flat-square) ![Downloads](https://img.shields.io/nuget/dt/Cartography.DynamicMap?style=flat-square)
+
+### **Description**
```

Comment:
Please check the guidelines for the structure of the REAME.md content.
https://dev.azure.com/nventive/Practice%20committees/_wiki/wikis/Committees.wiki/14813/Recipes?anchor=repository-structure-and-documentation

Reply:
There shouldn't be headers for Description and Goal.
You can add a short description just under the shields like it's done for other recipes.
Ex:
- https://github.com/nventive/MallardMessageHandlers#mallardmessagehandlers-
- https://github.com/nventive/MessageDialogService#message-dialog-service